mmv1/templates/terraform/examples/privateca_certificate_with_template.tf.erb

```diff
@@ -1,3 +1,4 @@
+# [START privateca_create_certificate_template]
```

Comment:
Are these notations adding value?  We don't have them for other resources, I don't believe.

Reply:
We would like to embed these samples in docs page. Hence enclosing with region tags.
Is there an alternative way to this?

Reply:
Could you simply put these tags in your documentation pages, rather than trying to put them in ours?

Reply:
Done. Thanks.